packages/cloudflare/src/cli/build/patches/plugins/instrumentation.ts

```diff
@@ -0,0 +1,84 @@
+import { existsSync } from "node:fs";
+import { join } from "node:path";
+
+import { type BuildOptions, getPackagePath } from "@opennextjs/aws/build/helper.js";
+
+import { patchCode } from "../ast/util.js";
+import type { ContentUpdater } from "./content-updater.js";
+
+export function patchInstrumentation(updater: ContentUpdater, buildOpts: BuildOptions) {
+  const builtInstrumentationPath = getBuiltInstrumentationPath(buildOpts);
+
+  return [
+    updater.updateContent(
+      "patch-load-instrumentation",
+      { filter: /\.(js|mjs|cjs|jsx|ts|tsx)$/, contentFilter: /async loadInstrumentationModule\(/ },
+      async ({ contents }) => patchCode(contents, getNext15Rule(builtInstrumentationPath))
+    ),
+    updater.updateContent(
+      "patch-prepareImpl",
+      { filter: /\.(js|mjs|cjs|jsx|ts|tsx)$/, contentFilter: /async prepareImpl\(/ },
+      async ({ contents }) => patchCode(contents, getNext14Rule(builtInstrumentationPath))
+    ),
+  ];
```

Comment:
We do not need to return 2 plugins, they are just "fake" plugins anyway

```suggestion
  updater.updateContent(
    "patch-instrumentation-next15",
    { filter: /\.(js|mjs|cjs|jsx|ts|tsx)$/, contentFilter: /async loadInstrumentationModule\(/ },
    async ({ contents }) => patchCode(contents, getNext15Rule(builtInstrumentationPath))
  );
  
  updater.updateContent(
    "patch-instrumentation-next14",
    { filter: /\.(js|mjs|cjs|jsx|ts|tsx)$/, contentFilter: /async prepareImpl\(/ },
    async ({ contents }) => patchCode(contents, getNext14Rule(builtInstrumentationPath))
  );

  return {
    "patch implementation",
    setup() {},
  };
```